huaweicloud/provider.go

```diff
@@ -542,6 +542,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_antiddos_quota":                        antiddos.DataSourceAntiDdosQuota(),
 
 			"huaweicloud_aom_access_codes":                    aom.DataSourceAomAccessCodes(),
+			"huaweicloud_aom_alarm_notified_history":          aom.DataSourceAlarmNotifiedHistory(),
 			"huaweicloud_aom_aggregation_metrics":             aom.DataSourceAggregationMetrics(),
 			"huaweicloud_aom_alarm_action_rules":              aom.DataSourceAomAlarmActionRules(),
 			"huaweicloud_aom_alarm_group_rules":               aom.DataSourceAlarmGroupRules(),
```

Comment:
```suggestion
			"huaweicloud_aom_aggregation_metrics":             aom.DataSourceAggregationMetrics(),
			"huaweicloud_aom_alarm_action_rules":              aom.DataSourceAomAlarmActionRules(),
			"huaweicloud_aom_alarm_group_rules":               aom.DataSourceAlarmGroupRules(),
			"huaweicloud_aom_alarm_notified_history":          aom.DataSourceAlarmNotifiedHistory(),
```
Reorder because the reference place of new data source is incorrect

Reply:
done